### PR TITLE
mocks expiration date on standardbounties

### PR DESCRIPTION
**Pull request by owocki**
##### Description
implements option 3 that @mbeylin suggested here => https://github.com/Bounties-Network/StandardBounties/issues/25

> the deadline you enter on gitcoin is stored off-chain (in the JSON object that's on IPFS), and the on-chain deadline is just max_int or something like that to ensure it never really expires

##### Checklist

- [X] linter status: 100% pass
- [X] changes don't break existing behavior
- [X] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)
* sync_geth
* frontend

##### Testing
* i have tested it 

##### Refers/Fixes
* https://github.com/Bounties-Network/StandardBounties/issues/25

### CODE REVIEW

**Comment by owocki** [comment]
@mbeacom can i get 👀 on this?

File: app/dashboard/utils.py [hunk 1]
```diff
@@ -172,6 +172,11 @@ def get_bounty(bounty_enum, network):
     if 'Failed to get block' in bounty_data_str:
         raise IPFSCantConnectException("Failed to connect to IPFS")
 
+    # https://github.com/Bounties-Network/StandardBounties/issues/25
+    override_deadline = bounty_data.get('payload', {}).get('expire_date', False)
```
Comment (mbeacom) [inline app/dashboard/utils.py]: Will the override ever be `False`?

File: app/assets/v2/js/pages/new_bounty.js [hunk 2]
```diff
@@ -264,11 +267,11 @@ $(document).ready(function(){
             var _paysTokens = !isETH;
             var bountyIndex = bounty.issueAndActivateBounty(
                 account,            // _issuer
-                expire_date,        // _deadline
+                mock_expire_date,   // _deadline
```
Comment (mbeacom) [inline app/assets/v2/js/pages/new_bounty.js]: In this case, we always want to use the `mock_expire_date` here?

File: app/assets/v2/js/pages/bounty_details.js [hunk 3]
```diff
@@ -345,7 +345,7 @@ pull_interest_list(result['pk'], function(is_interested){
         actions.push(entry);
     }
 
-    if(result['status']=='open' || result['status']=='started' || result['status']=='submitted' ){
+    if(result['status']=='open' || result['status']=='started' || result['status']=='submitted' ){ // TODO -- do we want to change how this functions if expired due to https://github.com/gitcoinco/web/pull/566 ?
```
Comment (mbeacom) [inline app/assets/v2/js/pages/bounty_details.js]: 🤔

File: app/dashboard/utils.py (hunk 1 above)
Comment (owocki) [reply app/dashboard/utils.py]: yes.. for bounties that were submitted on the bounties network or on gitcoin, but before this PR is merged

File: app/assets/v2/js/pages/new_bounty.js (hunk 2 above)
Comment (owocki) [reply app/assets/v2/js/pages/new_bounty.js]: i believe so... unless we want to provide the user the optional of actually expiring their bounty.. which i think might be too many explanatory overhead.. maybe pixelant will disagree

File: app/assets/v2/js/pages/bounty_details.js (hunk 3 above)
Comment (owocki) [reply app/assets/v2/js/pages/bounty_details.js]: i think the answer is that if theres a mock expiration date we want to display all these options even if the bounty is expired... sound okay to you?

**Pull request merged**